<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrescriptorMainWin</class>
 <widget class="QMainWindow" name="PrescriptorMainWin">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>923</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Prescriptor</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="Central">
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="PrescriptTab">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <attribute name="title">
        <string>Kê đơn thuốc</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="pre_nameLabel">
          <property name="text">
           <string>Họ và tên:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="pre_nameInput"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="pre_addressLabel">
          <property name="text">
           <string>Địa chỉ:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="pre_addressInput"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="pre_birthLabel">
          <property name="text">
           <string>Ngày sinh:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QWidget" name="birthAndPhoneNumContainer" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="pre_birthInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="inputMask">
              <string>00/00/0000</string>
             </property>
             <property name="text">
              <string>01/01/2000</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pre_YearsOld">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pre_GenderLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Giới tính</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="pre_ifFemale">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Nữ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pre_phoneNumberLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Số điện thoại:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pre_phoneNumberInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="pre_visionLabelLeft">
          <property name="text">
           <string>Thị lực:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QWidget" name="pre_visionContainer" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="horizontalSpacing">
            <number>3</number>
           </property>
           <property name="verticalSpacing">
            <number>2</number>
           </property>
           <item row="3" column="0">
            <widget class="QLabel" name="pre_OSLabel">
             <property name="text">
              <string>MT:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="pre_ODLabel">
             <property name="text">
              <string>MP:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="pre_visionLabelRight">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>TL:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="pre_leftNoGlassInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="pre_noGlassVisonLabel">
             <property name="text">
              <string>Không kính (X/10)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="pre_rightNoGlassInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="pre_rightGlassDioptreInput"/>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="pre_rightGlassVisionInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="pre_leftGlassDioptreInput"/>
           </item>
           <item row="3" column="3">
            <widget class="QLineEdit" name="pre_leftGlassVisionInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLineEdit" name="pre_rightEyePressureInput"/>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="pre_eyePressureLabel">
             <property name="text">
              <string>Nhãn áp (mmHg)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QLineEdit" name="pre_leftEyePressureInput"/>
           </item>
           <item row="0" column="2" colspan="2">
            <widget class="QLabel" name="pre_glassVisionLabel">
             <property name="text">
              <string>Có kính (Dioptre, X/10)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="pre_diagnosisLabel">
          <property name="text">
           <string>Chẩn đoán:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="pre_diagnosisInput"/>
        </item>
        <item row="11" column="1">
         <widget class="QTextEdit" name="pre_diagnosisListInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="pre_drugsLabel">
          <property name="text">
           <string>Tên thuốc:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QWidget" name="pre_drugsContainer" native="true">
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="1" column="3">
            <widget class="QLabel" name="pre_drugUsageLabel">
             <property name="text">
              <string>HDSD:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="pre_drugHowmanyLabel">
             <property name="text">
              <string>SL:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="pre_drugsInput"/>
           </item>
           <item row="1" column="4">
            <widget class="QLineEdit" name="pre_drugUsageInput"/>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="pre_drugHowmanyInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QTableWidget" name="pre_drugsView"/>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="pre_noteLabel">
          <property name="text">
           <string>Ghi chú:</string>
          </property>
         </widget>
        </item>
        <item row="16" column="1">
         <widget class="QWidget" name="pre_noteContainer" native="true">
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="horizontalSpacing">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="3" column="0">
            <widget class="QCheckBox" name="pre_revisitCheckbox">
             <property name="text">
              <string>Tái khám sau</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="pre_noteInput">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="pre_revisitperiodInput"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QCheckBox" name="pre_warnNoPhoneCheckbox">
          <property name="text">
           <string>Hạn chế tiếp xúc với các thiết bị điện tử như TV, iPad (tablet), điện thoại thông minh,...</string>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QLabel" name="pre_fillerLabel1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="QWidget" name="PrescriptFuncBtnsContainer" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="2">
            <widget class="QPushButton" name="pre_removePreBtn">
             <property name="text">
              <string>Làm sạch thông tin</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pre_printPreBtn">
             <property name="text">
              <string>In đơn thuốc</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pre_savePreBtn">
             <property name="text">
              <string>Lưu đơn thuốc</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="pre_resultLabel">
          <property name="text">
           <string>KQ khám:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="pre_resultInput"/>
        </item>
        <item row="9" column="1">
         <widget class="QTextEdit" name="pre_resultListInput"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="pre_stateLabel">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SearchTab">
       <attribute name="title">
        <string>Tìm đơn thuốc</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="searchDataInputContainer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QLineEdit" name="search_nameOrPhoneNumInput"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="search_nameOrPhoneNumLabel">
             <property name="text">
              <string>Họ tên/SĐT:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="search_searchBtn">
             <property name="text">
              <string>Tìm kiếm</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="search_patientInfoView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="search_patientTableView">
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="SearchFuncBtnsContainer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="search_printPreBtn">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>In đơn thuốc</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="search_editPreBtn">
             <property name="text">
              <string>Sửa lại đơn thuốc</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="search_removePreBtn">
             <property name="text">
              <string>Xoá đơn thuốc</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="search_inheritOldPreBtn">
             <property name="text">
              <string>Kế thừa đơn thuốc cũ</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PropertiesTab">
       <attribute name="title">
        <string>Cài đặt</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Thông tin:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="prop_recbacState">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="prop_nameLabel">
          <property name="text">
           <string>Tên cửa hàng:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="prop_nameInput"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="prop_addrLabel">
          <property name="text">
           <string>Địa chỉ cửa hàng:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="prop_addrInput"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="prop_phoneNumLabel">
          <property name="text">
           <string>SĐT liên lạc:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="prop_phoneNumInput"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="prop_doctorLabel">
          <property name="text">
           <string>Bác sĩ kê đơn:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="prop_doctorInput"/>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="prop_saveInfo">
          <property name="text">
           <string>Lưu thông tin</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Dữ liệu bệnh nhân:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QWidget" name="prop_recbacContainer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="3">
            <widget class="QPushButton" name="prop_recoverBtn">
             <property name="text">
              <string>Khôi phục</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="prop_backupLabel">
             <property name="text">
              <string>Sao lưu dữ liệu:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="prop_recoverLabel">
             <property name="text">
              <string>Khôi phục dữ liệu:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="prop_backupBtn">
             <property name="text">
              <string>Sao lưu</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="prop_currentState">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
